python/cuml/dask/preprocessing/label.py

```diff
@@ -32,59 +32,46 @@ class LabelBinarizer(BaseEstimator):
     Examples
     --------
 
-    Examples
-    --------
-
     Create an array with labels and dummy encode them
 
     .. code-block:: python
 
-        import cupy as cp
-        import cupyx
-        from cuml.dask.preprocessing import LabelBinarizer
-
-        from dask_cuda import LocalCUDACluster
-        from dask.distributed import Client
-        import dask
-
-        cluster = LocalCUDACluster()
-        client = Client(cluster)
-
-        labels = cp.asarray([0, 5, 10, 7, 2, 4, 1, 0, 0, 4, 3, 2, 1],
-                            dtype=cp.int32)
-        labels = dask.array.from_array(labels)
+        >>> import cupy as cp
+        >>> import cupyx
+        >>> from cuml.dask.preprocessing import LabelBinarizer
 
-        lb = LabelBinarizer()
+        >>> from dask_cuda import LocalCUDACluster
+        >>> from dask.distributed import Client
+        >>> import dask
 
-        encoded = lb.fit_transform(labels)
+        >>> cluster = LocalCUDACluster()
+        >>> client = Client(cluster)
 
-        print(str(encoded.compute())
-
-        decoded = lb.inverse_transform(encoded)
-
-        print(str(decoded.compute())
-
-
-    Output:
-
-    .. code-block::
+        >>> labels = cp.asarray([0, 5, 10, 7, 2, 4, 1, 0, 0, 4, 3, 2, 1],
+        ...                     dtype=cp.int32)
+        >>> labels = dask.array.from_array(labels)
 
+        >>> lb = LabelBinarizer()
+        >>> encoded = lb.fit_transform(labels)
+        >>> print(encoded.compute())
```

Comment:
Why skip not needed here unlike the others?

Reply:
This example on labels is easily reproducible because there's no numerical imprecision involved. Unlike most dask algorithm.